### ISSUE TITLE
Proposal: Multiple crumbs per route

### ISSUE THREAD

**Issue by mike-north** [poteto/ember-crumbly#19]
TL;DR - I have a use case where it would be nice to have a certain route provide more than one "crumb" in the breadcrumb trail. 
### The Use Case

Here's the practical use case (being verbose for clarity's sake)

Let's say I have my routes set up as follows: 

**app/router.js**

``` js
this.route('companies', {path: '/companies'}, function () {
    // companies
    this.route('index', {path: '/'} );
});

this.route('company', {path: '/company/:id'}, function () {
    // company/:id/departments
    this.route('departments', {path: '/departments'});
});

this.route('department', {path: '/department/:id'}, function () {
    // department/:id/employees
    this.route('employees', {path: '/employees'});
});

this.route('employee', {path: '/employee/:id'}, function () {
    // employee/:id/documents
    this.route('documents', {path: '/documents'});
});
```

I'd argue that this is not an unconventional way to set things up. The principle that I'm sticking to here is that, with the exception of the `companies.index` route, each leaf level route is a list of items under an addressable resource. That being said, conceptually, the hierarchy is clearly still

```
Company
    Department
          Employee
               Document
```
### What I want to be able to do

On the page I land on for URL `/employee/poteto/documents`, I would like to be able to build the breadcrumb trail like this (all potentially linkable)

```
Clients  >  Dockyard  >  Engineering  >  Lauren  >  Documents
```

Despite the active routes being

```
application
    employee
        employee.documents
```
### A proposed solution

a route can provide multiple 'crumbs' to the breadcrumb trail
- **application** - No breadcrumbs provided 
  - **employee** - `Clients > Dockyard > Engineering > Lauren`
    - **employee.documents** - `> Documents`

I'm not sure how to best add this to the API surface you already have defined (I really like it for the use cases that this add-on is designed for), but if we consider that an implementation detail, I think that decoupling model relationship hierarchies from route hierarchies would make the addon a bit more flexible.

Thoughts?

**Comment by jchamet** [comment]
+1

**Comment by abhilashlr** [comment]
+1

**Comment by ssured** [comment]
:+1:

**Comment by jme783** [comment]
+1

**Comment by alvinvogelzang** [comment]
I created this PR for it: https://github.com/poteto/ember-crumbly/pull/85 

Any feedback would be nice :-)

**Comment by dguettler** [comment]
#125 has been merged and published as 1.0.8
